e2e/package.json

```diff
@@ -15,6 +15,7 @@
   "author": "Benjamin Rosas <ben@aliencyb.org>",
   "devDependencies": {
     "cypress": "^6.2.1",
+    "cypress-iframe": "^1.0.1",
```

Comment:
on npm this is the latest version. looks good. https://www.npmjs.com/package/cypress-iframe